avocado/utils/network/interfaces.py

```diff
@@ -128,6 +128,74 @@ def add_ipaddr(self, ipaddr, netmask):
         except Exception as ex:
             raise NWException("Failed to add address {}".format(ex))
 
+    @property
+    def vlans(self):
+        """Return a list of vlans"""
+        vlans = {}
+        with open('/proc/net/vlan/config', encoding="utf-8") as vlan_config_file:
```

Comment:
I think this deserves to be protected where there are no configured in the system.  Snippet example is:

```python
from avocado.utils.network.interfaces import NetworkInterface
from avocado.utils.network.hosts import LocalHost

n = NetworkInterface('enp1s0', LocalHost)
n.vlans
```

This would cause a crash:

```
Traceback (most recent call last):
  File "/tmp/vlan.py", line 5, in <module>
    n.vlans
  File "/home/cleber/src/avocado/avocado/utils/network/interfaces.py", line 135, in vlans
    with open('/proc/net/vlan/config', encoding="utf-8") as vlan_config_file:
FileNotFoundError: [Errno 2] No such file or directory: '/proc/net/vlan/config'
```

IMO, this should simply return an empty dict.